wrappers/obj-c/ODWPrivacyGuard.h

```diff
@@ -71,6 +71,11 @@ typedef NS_ENUM(NSUInteger, ODWDataConcernType)
  */
 +(void)addIgnoredConcern:(NSString *) EventName withNSString:(NSString *)FieldName withODWDataConcernType:(ODWDataConcernType)IgnoredConcern;
 
+/*!
+ @brief Reset the Privacy Guard instance. This should be used after LogManager::FlushAndTeardown is called.
```

Comment:
Not sure I follow, if FlushAndTearDown is usually last interaction client has with 1DS, why a reset is needed? If that is required, wouldn't be better to do that as part of FlushAndTearDown automatically?

Reply:
Sid clarified offline scenario. This is targeted to Teams, so better to keep it decoupled.

Reply:
Discussed offline. We want to avoid strong dependency between LogManager and PrivacyGuard, and as such are going with the separate API call approach